### DISCUSSION TITLE
Registro da Python Nordeste 2023

### DISCUSSION THREAD

**Discussion by cecivieira** [apyb/comunidade#77] [Geral]
Boa tarde, @diretoria/apyb !

Nesse ano temos uma comissão responsável pela Python Nordeste, ou seja, não existe 1 big kahuna, mas um grupo. Como devemos registrar o evento junto a APyB?

**Comment by rougeth** [comment]
Oi Ceci! Para a APyB, não importa como o evento será organizado. O único requisito é ter uma pessoa, associada a APyB, que representará o evento perante a Associação até a realização do evento.

**Comment by cecivieira** [comment]
Entendi.

A pergunta é mais no sentido de saber se mais de uma pessoa pode representar junto à APyB.
